### DISCUSSION TITLE
Antenna Separation

### DISCUSSION THREAD

**Discussion by TerraKate** [emanuelhuber/RGPR#64] [General]
Hello @emanuelhuber , thank you very much for your work with RGPR!

I am using this to create a custom script for GPR processing. The script runs from the terminal and I am wondering how to remove the "Antenna Separation" message for user readability. I want the user to have the option to set the antenna separation but when I receive the input and change the @antsep value, it doesn't seem to hold or remove the console message. I may be setting the value wrong. Any help would be appreciated,

**Comment by emanuelhuber** [comment]
Hello @TerraKate 
Thank you for contacting me. 

Do you mean the "Antenna separation..." when reading that shows up when reading GPR data? You can suppress this message by setting `verbose = FALSE` in the function `readGPR()`.

```
x <- readGPR(dsn = "XLINE00.DT1", verbose = FALSE)
```

To set the antenna separation, use the function

```
antsep(x) <- 1   # 1 meter
```

I do not know how you wrote and run your script but there is the possibility to pass a list of file paths (or file connections) to the function `readGPR()` through the argument `dsn` (so you can read the GPR data as well as the associated GPS data and marker data, if they exist for your data format).


Let me know if it solves your problem. Do not hesitate to contact me if you have any question!

Best,

Emanuel

**Reply by TerraKate** [reply]
Hello @emanuelhuber,

Thank you for your response! I must have missed the verbose parameter, I knew it would be a simple fix that I wasn't seeing. This works well as I am already passing in a file path with `dsn`. Great work, 

Terra
